### ISSUE TITLE
HTMD: Logging setup failed

### ISSUE THREAD

**Issue by Lizeerd** [BenevolentAI/DeeplyTough#17]
HI, there
after following the code setup,  i tried to eval on custom dataset, and it raised: 
HTMD: Logging setup failed
can someone show me how to fix it

**Comment by JoshuaMeyers** [comment]
Hey @Lizeerd thanks for your interest in deeplytough. Could you post the full stack trace, usually failing to setup logging is just a warning message and doesn't stop the program running

**Comment by Lizeerd** [comment]
hi, thanks for ur reply, and the full stack trace is as follows:
HTMD: Logging setup failed
Total number of parameters: 390512
VoxelNetwork(
  (0): GatedBlock(
    (gate_act): ScalarActivation()
    (conv): SE3BNConvolution (SE3Kernel ([(8, 0)] -> [(16, 0), (16, 1), (16, 2), (16, 3), (48, 0)], size=7), eps=1e-05, momentum=0.1)
  )
  (1): GatedBlock(
    (scalar_act): ScalarActivation()
    (gate_act): ScalarActivation()
    (conv): SE3BNConvolution (SE3Kernel ([(16, 0), (16, 1), (16, 2), (16, 3)] -> [(32, 0), (32, 1), (32, 2), (32, 3), (96, 0)], size=3), eps=1e-05, momentum=0.1)
  )
  (2): GatedBlock(
    (scalar_act): ScalarActivation()
    (gate_act): ScalarActivation()
    (conv): SE3BNConvolution (SE3Kernel ([(32, 0), (32, 1), (32, 2), (32, 3)] -> [(48, 0), (48, 1), (48, 2), (48, 3), (144, 0)], size=3), eps=1e-05, momentum=0.1)
  )
  (3): GatedBlock(
    (scalar_act): ScalarActivation()
    (gate_act): ScalarActivation()
    (conv): SE3BNConvolution (SE3Kernel ([(48, 0), (48, 1), (48, 2), (48, 3)] -> [(64, 0), (64, 1), (64, 2), (64, 3), (192, 0)], size=3), eps=1e-05, momentum=0.1)
  )
  (4): GatedBlock(
    (conv): SE3BNConvolution (SE3Kernel ([(64, 0), (64, 1), (64, 2), (64, 3)] -> [(256, 0)], size=3), eps=1e-05, momentum=0.1)
  )
  (5): ReLU(inplace)
  (6): BatchNorm3d(256, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
  (7): Conv3d(256, 128, kernel_size=(1, 1, 1), stride=(1, 1, 1))
)
8it [00:00, 333.69it/s]
INFO:engine.datasets:Dataset size: 8
  0%|                                                            | 0/1 [00:00<?, ?it/s]/root/miniconda3/envs/deeplytough/lib/python3.6/site-packages/torch/utils/checkpoint.py:21: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")
/root/miniconda3/envs/deeplytough/lib/python3.6/site-packages/torch/nn/functional.py:1332: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
100%|████████████████████████████████████████████████████| 1/1 [00:17<00:00, 17.00s/it]
7it [00:00, 16834.94it/s]
Evaluation finished, see /root/DeeplyTough/results/Custom-DeeplyTough-networks.csv

**Comment by JoshuaMeyers** [comment]
Hey @Lizeerd, this looks pretty much as it should. Can you see the CSV file with the results in it?

**Comment by Lizeerd** [comment]
hi @JoshuaMeyers , the csv file is as follows, and i think it goes well. I'm new in this field,  sorry for this dummy issue, and thanks a lot for your help.
['1a9t/1a9t_clean', '1a9t/1a9t_site_1', '1a05B/1a05B', '1a05B/pockets/pocket0_atm', '-0.22254647314548492']
['1a9t/1a9t_clean', '1a9t/1a9t_site_1', '1a05B/1a05B', '1a05B/pockets/pocket1_atm', '-0.5968486070632935']
['1a9t/1a9t_clean', '1a9t/1a9t_site_1', '1a05B/1a05B', '1a05B/pockets/pocket2_atm', '-0.12686017155647278']
['1a9t/1a9t_clean', '1a9t/1a9t_site_1', '1a05B/1a05B', '1a05B/pockets/pocket3_atm', '-0.7348098158836365']
['1a9t/1a9t_clean', '1a9t/1a9t_site_1', '1a05B/1a05B', '1a05B/pockets/pocket4_atm', '-0.4152827560901642']
['1a9t/1a9t_clean', '1a9t/1a9t_site_1', '1a05B/1a05B', '1a05B/pockets/pocket5_atm', '-0.6835595965385437']
['1a9t/1a9t_clean', '1a9t/1a9t_site_1', '1a05B/1a05B', '1a05B/pockets/pocket6_atm', '-0.6399115324020386']
